common/src/main/java/org/apache/falcon/entity/EntityUtil.java

```diff
@@ -1082,4 +1091,15 @@ public static JOBPRIORITY getPriority(Process process) {
         }
         return JOBPRIORITY.NORMAL;
     }
+
```

Comment:
I understand it is a simple and straightforward method but since it is a public utility method, it will be useful to add a bit of documentation on what this method does and when to use it?